server/index.js

```diff
@@ -42,6 +42,11 @@ if (config.dev) {
   builder.build();
 }
 
+// explicit cache control on sdk resources
+app.use('/sdk', (req, res, next) => {
+  res.setHeader('Cache-Control', 'public, max-age=3600, s-maxage=3600, stale-if-error=604800, stale-while-revalidate=604800');
```

Comment:
For reference, the Facebook SDK entry point use this: `Cache-Control: public,max-age=1200,stale-while-revalidate=3600`.

Reply:
we just copycat 1200 and 3600 then